components/email/EmailCaptureHelper.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+export default function EmailCaptureHome({ children, data }) {
```

Comment:
```suggestion
export default function EmailCaptureHelper({ children, data }) {
```

@audreyfeldroy says:
Either change EmailCaptureHome to match the name of the file, or rename the file to `EmailCaptureHome`.